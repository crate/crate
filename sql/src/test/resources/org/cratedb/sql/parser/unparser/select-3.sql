SELECT DISTINCT MAX(x),y||y AS yy FROM t WHERE z > 3.14 AND w IS NOT NULL OR y LIKE 'f%' GROUP BY yy HAVING yy > 'z' ORDER BY yyo NULLS FIRST