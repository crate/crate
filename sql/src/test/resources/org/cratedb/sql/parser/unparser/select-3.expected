SELECT DISTINCT MAX(x), (y || y) AS yy FROM t WHERE ((z > 3.14) AND (NOT (w IS NULL))) OR (y LIKE 'f%') GROUP BY yy HAVING yy > 'z' ORDER BY yyo NULLS FIRST